src/backend/aspen/app/views/phylo_trees.py

```diff
@@ -37,6 +37,8 @@ def humanize_tree_name(s3_key: str):
     basename = re.sub(r".json", "", json_filename)
     if basename == "ncov_aspen":
         return s3_key.split("/")[1]  # Return the directory name.
+    if basename == "ncov_aspen_3m":
+        return s3_key.split("/")[1] + " recency-focused build"  # Name it differently from overview tree.
```

Comment:
I don't really like this hack, but it will work if we're in a super-hurry. I'll think of a cleaner option.

Reply:
We can go without re-naming the trees if this evolves into a full-on tree name tree table update XD Closing this PR!

Reply:
I think we just want to add a `--tree-name` flag to the create-phylo-runs script, and put the name you want here, and then we'll be all set:
https://github.com/chanzuckerberg/aspen/blob/trunk/src/backend/aspen/workflows/nextstrain_run/run_nextstrain_scheduled.sh#L35

I'll give this a shot.